### ISSUE TITLE
[Feature Request]: Restore all opened workflows on page reload

### ISSUE THREAD

**Issue by huchenlei** [Comfy-Org/ComfyUI_frontend#1575]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Currently, only the last active workflow was restored, while all other opened workflows states are lost. We should restore the opened workflows and their states as well just in case there are unsaved changes made to these workflows.

### Proposed workflow

- User opens multiple workflows.
- User refreshes the page
- The opened workflows list remain the same after the page reload.

┆Issue is synchronized with this [Notion page](https://www.notion.so/Feature-Request-Restore-all-opened-workflows-on-page-reload-15b6d73d36508139b26adc960a11cf5a) by [Unito](https://www.unito.io)

**Comment by LukeG89** [comment]
Will this also be applied when quitting ComfyUI? On next startup all previously opened workflows will be loaded?

**Comment by huchenlei** [comment]
> Will this also be applied when quitting ComfyUI? On next startup all previously opened workflows will be loaded?

Yes. It should. The current method is storing via browser localStorage which should be the same as long as you start the ComfyUI server on the same host and port.

We are also considering if it is time now to add a DB for persistece on server side.

**Comment by LukeG89** [comment]
> We are also considering if it is time now to add a DB for persistece on server side.

I agree about that, ComfyUI should stop relying on browser storage

**Comment by mijuku233** [comment]
I always use refresh to close other workflows. If this feature is implemented, I hope to add a shortcut key to close inactive workflows.

**Comment by JorgeR81** [comment]
> I always use refresh to close other workflows. If this feature is implemented, I hope to add a shortcut key to close inactive workflows.

Yes, I would also like to have this. 

A shortcut key to close all inactive workflows at a time.

We can use `Shift+click` to close a workflows tab, without the warning, but only one at a time.

And also on Windows 10, pressing `Shift` to many time, enables the "sticky keys" feature.
We coul also have an option to change it to `Ctrl+click`.

Or an user option, to disable all workflow save warnings.

**Comment by webfiltered** [comment]
You can disable sticky keys shortcut, if you're not using it.  Or just keep shift held down and click many times.

imo it would be ideal to mimic popular behaviour so it is intuitive - middle click closes tabs.

A context menu option to close other tabs would be good.  Implementing all these as UI commands that can have configurable shortcuts is also good.
